cargo-concordium/src/build.rs

```diff
@@ -33,6 +33,7 @@ use std::{
 /// Encode all base64 strings using the standard alphabet and padding.
 const ENCODER: base64::engine::GeneralPurpose = general_purpose::STANDARD;
 
+/// Convert a string to snake case by replacing `-` with `_`.
```

Comment:
Maybe the functionality is clear from the code, but not the purpose.

Maybe add that this is what rust does as well when converting crate names (which often have `-`) to module names (which cannot have that).